common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1890,7 +1883,7 @@ def copy_all_asset_metadata(self, source_course_key, dest_course_key, user_id):
         """
         source_assets = self._find_course_assets(source_course_key)
         dest_assets = {'assets': source_assets.asset_md.copy(), 'course_id': unicode(dest_course_key)}
-        self.asset_collection.remove({'course_id': unicode(dest_course_key)})
+        self.asset_collection.delete_many({'course_id': six.text_type(dest_course_key)})
```

Comment:
@pkulkark The change from `unicode` to `six.text_type` is not in the original commit (9b69d5a62684dc3d9d86b7e71081bebd432bd7d9). We should revert it or [`import six`](https://github.com/open-craft/edx-platform/blob/9b69d5a62684dc3d9d86b7e71081bebd432bd7d9/common/lib/xmodule/xmodule/modulestore/mongo/base.py#L25) at the top of this file.

Reply:
@itsjeyd Nice catch! Oddly it shows up on the [PR](https://github.com/open-craft/edx-platform/pull/196/files#diff-3f177df23ae21e4e951d893e0818d864L1912) but not on the commit. Anyway I've reverted it back to `unicode`.